frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/propertiesList.component.tsx

```diff
@@ -34,6 +34,9 @@ interface PropertiesListProps {
 export const PropertiesList = ({ module, properties, onPropertyBlur }: PropertiesListProps) => {
 	const { formState } = useFormContext();
 	const isReadOnly = TicketsCardHooksSelectors.selectReadOnly();
+
```

Comment:
The bottom padding got lost also the line separating

<img width="332" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/f367d33d-9516-4207-8af0-87048e3efcd3">
 
take a lok at the padding here

![image](https://github.com/3drepo/3drepo.io/assets/11945337/65dc6364-0a41-4fb4-a94d-33ca9e5af6f6)

[original mockup](https://www.figma.com/file/3FjNEOdQbcHwzquVZxUqMl/3D-Repo?type=design&node-id=340-379&mode=design&t=xlhQ35jN4b2bmZCV-0)